tests/7-Input.test.tsx

```diff
@@ -0,0 +1,34 @@
+import "@testing-library/jest-dom/extend-expect";
+
+import { InputProps } from "common/Input";
+import { Password } from "stories/7-Input.stories";
+import React from "react";
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+jest.mock("@storybook/addon-actions");
+
+describe("Password", () => {
```

Comment:
Someone in fandom coders suggested this style for writing `describe\test`, which I really like: https://github.com/BobaBoard/boba-components/blob/4180ef9aa93a8d2b16713e4b53f9a6b499919825/tests/05-Posts/03-PostFooter.test.tsx#L23

I think this test could benefit from something similar.

Note that `it` and `test` should be equivalent.

Reply:
.. I tried